core/storage/suggestion/gae_models.py

```diff
@@ -484,13 +484,43 @@ def get_in_review_translation_suggestions(
             list(SuggestionModel). A list of suggestions that are of the given
             type, which are in review, but not created by the given user.
         """
+
         return cls.get_all().filter(datastore_services.all_of(
             cls.status == STATUS_IN_REVIEW,
             cls.suggestion_type == feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
             cls.author_id != user_id,
             cls.language_code.IN(language_codes)
         )).fetch(feconf.DEFAULT_SUGGESTION_QUERY_LIMIT)
 
+    @classmethod
+    def get_in_review_translation_suggestions_with_exp_ids(
+            cls, user_id: str, language_codes: List[str],
+            exp_ids: List[str]
+    ) -> Sequence[GeneralSuggestionModel]:
+        """Gets all translation suggestions which are in review
```

Comment:
...suggestions for the given language codes which are ...

Reply:
Done. 

Sorry, missed this since language codes was not part of the docstring earlier either, so I thought there might be a reason why it was skipped and didn't specify either.